.github/workflows/gh-pages.yml

```diff
@@ -22,23 +22,9 @@ jobs:
           node-version: 20
           registry-url: "https://registry.npmjs.org"
 
-      - name: Run typedoc
+      - name: Build website
         run: |
-          npm install 
-          npm run bootstrap
-          npm run build
-          npm run typedoc
-
-      - name: Fix typedoc links
-        run: |
-          npm install
-          npm run setup-typedoc
-        working-directory: website
-
-      - name: Build 🔧
-        run: |
-          npm run build
-        working-directory: website
+          ./.github/scripts/website.sh
```

Comment:
Is extracting this into a separate script really necessary? Feels like we're just creating more files for ourselves to look through.

Or is the `.github/scripts/website.sh` script used in multiple places? Or needed for Netlify or something?

Reply:
I suppose I'm a bit confused here. We have this `gh-pages` workflow, but we're also deploying the website to Netlify? Why do we need both?

Reply:
I added it cause it was needed for Netlify, no other reasons

Reply:
So why is `gh-pages` workflow needed if we deploy from Netlify?

Reply:
Netlify is only for preview! not sure how it'd work just deploying there @baumstern there might be costs involved/and/or an account to setup?

Reply:
Ahhhh, I gotcha. Ok that seems silly lol, but not a change for this PR. Let's resolve that separately.

Reply:
Having a build script offers two benefits: it simplifies the gh-pages.yml and makes the build step more universally accessible to the community, regardless of knowledge of GitHub Actions' syntax

Reply:
> Netlify is only for preview! not sure how it'd work just deploying there @baumstern there might be costs involved/and/or an account to setup?

It's a free tier :)

Reply:
> Having a build script offers two benefits: it simplifies the gh-pages.yml and makes the build step more universally accessible to the community, regardless of their knowledge of GitHub Actions syntax

oh wow now we can building website just using `npm run build` command thanks to this PR: https://github.com/privacy-scaling-explorations/maci/pull/888

Reply:
so well now to get it working we have to build twice the website (check the setup-website.sh script) - first time we build with lerna, then have to build individually 🤔 
any suggestions @0xmad @samajammin @baumstern ?
Also I had to make the `setupSolidityDocs` and `setupTypedoc` silently complete without making any changes if either typedoc or the solidity output dirs are not present.. which might result in a user locally skipping some steps (run typedoc, and run solidity docgen) thought the website would still work, just it would miss content on typedoc and solidity docs.. curious about what you think here

Reply:
I think it's okay because the build time takes about 2 to 3 minutes

Reply:
not sure I follow you! how does that make it ok?